### PR TITLE
Mtran's project

### PR DESCRIPTION
**Pull request by Msiutsz**

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let x=1;
+let y=2;
+let z=3;
+
+if ((x+y)==z) {
+  console.log(z);
+}
+
+if (x===w) {
+  console.log (true);
```
Comment (MadEste) [inline calculator.js]: Good check with Strict Equality

File: countdown.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let x;
+
+for (x=10; x>=0; x--) {
```
Comment (MadEste) [inline countdown.js]: Nice work. You can also define x inside the for loop. `let x = 10; x >= 0;.....`

File: lesson03/exercise-one.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+console.log(rainbowColors);
+
+rainbowColors.splice(2, 1);
+
+console.log(rainbowColors);
+
+const rainbowColorsList = rainbowColors.join (', ');
+
+console.log(rainbowColors.join (', '));
```
Comment (MadEste) [inline lesson03/exercise-one.js]: 👍 Could also log rainbowColorsList

File: lesson03/exercise-two.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+let sentence='Bacon and eggs are delicious!.'
+let words=sentence.split(' ');
+console.log(words[2]);
```
Comment (MadEste) [inline lesson03/exercise-two.js]: 👍

**Comment by Msiutsz** [comment]
Added lesson 4

File: lesson04/index.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <h1>Square it!</h1>
+    <button onclick="numberSquared()">Start</button>
+    <h1>Multiply it!</h1>
+    <button onclick="numberMultiply()">Start</button>
+    <script src="./assignment.js" charset="utf-8">
```
Comment (MadEste) [inline lesson04/index.html]: 👍

File: lesson05/banking.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+let response = "";
+let balance = 10000.00;
+let count = 0;
+
+do {
+
+  response = prompt("Welcome to your bank, please select an option\n(Q)uit\n(W)ithdraw\n(B)alance\n(D)eposit");
+
+  switch(response.toLowerCase()) {
```
Comment (MadEste) [inline lesson05/banking.js]: Great work accepting both upper and lower cases

File: lesson05/banking.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
+function startBank () {
+
+let response = "";
+let balance = 10000.00;
+let count = 0;
+
+do {
```
Comment (MadEste) [inline lesson05/banking.js]: Nice use of do while loop!

File: lesson05/banking.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
+function startBank () {
+
+let response = "";
+let balance = 10000.00;
+let count = 0;
```
Comment (MadEste) [inline lesson05/banking.js]: I do not see count used anywhere.

File: lesson05/banking.js [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+                +"New Balance: $" + balance);
+      } else if (deposit < 0.01) {
+        alert("The dollar amount entered is not valid")
+      } else if (deposit >= 50000.0) {
+        alert("The dollar amount is greater the the $50,000 deposit limit");
+      } else {
+        alert("Please enter a valid dollar amount");
+      }
+      break;
+      default:
```
Comment (MadEste) [inline lesson05/banking.js]: good use of default to break out on cancel

**Pull request closed without merging**